packages/grafana-ui/src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,75 @@
+import { css } from '@emotion/css';
+import React, { useCallback } from 'react';
+
+import { GrafanaTheme2, ThemeSpacingTokens } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+
+export type AlignItems =
+  | 'stretch'
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'baseline'
+  | 'start'
+  | 'end'
+  | 'self-start'
+  | 'self-end';
+
+export type JustifyContent =
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evenly'
+  | 'start'
+  | 'end'
+  | 'left'
+  | 'right';
+
+export type Direction = 'row' | 'row-reverse' | 'column' | 'column-reverse';
+
+export type Wrap = 'nowrap' | 'wrap' | 'wrap-reverse';
```

Comment:
All of these types are already defined by `CSSProperties['alignItems']`


`import React, { CSSProperties, useCallback } from 'react';`

Reply:
Maybe we can keep these for now, to make life a little bit simpler? `CSSProperties['alignItems']` will make us add custom argTypes anywhere it is used. https://github.com/storybookjs/storybook/issues/23900